### PR TITLE
add single node query e2e test

### PR DESCRIPTION
**Pull request by sonyafenge**
run example:
```
sonyali@sonyaowndev1:~/go/src/global-resource-service$ go run test/e2e/singleNodeQuery.go -v=9 --service_url=34.173.200.180:8080 --single_node_num=100 --batch_node_num=100 > ~/logs/nodeuery.log.2022-0907 2>&1 &
```

Verified on 5M nodes with single node query per second. you can find results at: sonyadev4:/home/sonyali/grs/logs/5000000/Daily/090922-190130

Metrics Item | Query Interval | Node Num for each interval | perc50 | perc90 | perc99
-- | -- | -- | -- | -- | --
[Metrics][Nodes] | 1s | 1 | 21.344525ms | 21.910167ms | 22.068838ms.
[Metrics][Nodes] | 1m0s | 0 | 0s | 0s | 0s.

### CODE REVIEW

**Review by q131172019 — COMMENTED** [review]
It is recommended to add inquiry test result for 5 million node. Generally I am fine with this PR.

File: resource-management/pkg/clientSdk/rest/request.go [hunk 1]
```diff
@@ -24,10 +24,9 @@ import (
 	"encoding/hex"
 	"encoding/json"
 	"fmt"
-	"global-resource-service/resource-management/pkg/clientSdk/rest/watch"
+	versioned "global-resource-service/resource-management/pkg/clientSdk/rest/watch"
```
Comment (yb01) [inline resource-management/pkg/clientSdk/rest/request.go:27]: imports group local packages

File: resource-management/pkg/clientSdk/rmsclient/rmsClient.go [hunk 2]
```diff
@@ -174,3 +175,29 @@ func (c *rmsClient) Watch(clientId string, versionMap types.TransitResourceVersi
...
+	req = req.Param("resourcePartition", rpName)
+	
+	respRet, err := req.DoRaw()
+	if err != nil {
+		return nil, err
+	}
+
+	resp := apiTypes.NodeResponse{}
+
+	err = json.Unmarshal(respRet, &resp)
```
Comment (yb01) [inline resource-management/pkg/clientSdk/rmsclient/rmsClient.go:197]: error check

File: resource-management/pkg/clientSdk/rest/request.go (hunk 1 above)
Comment (sonyafenge) [reply resource-management/pkg/clientSdk/rest/request.go]: fixed

File: resource-management/pkg/clientSdk/rmsclient/rmsClient.go (hunk 2 above)
Comment (sonyafenge) [reply resource-management/pkg/clientSdk/rmsclient/rmsClient.go:201]: fixed

**Review by yb01 — APPROVED** [review]
/lgtm

**Pull request merged**